KindleReadingTime/BookCell.swift

```diff
@@ -39,13 +41,14 @@ class BookCell: UITableViewCell {
     backgroundColor = .yellow
 
     addSubview(coverImageView)
-    coverImageView.frame = CGRect(x: 8, y: 8, width: 50, height: 64 )
```

Comment:
Basically the anchor constraints allow us to not have to define a height, because we anchor the bottom within its container to be always 8 from the edge.. neat!